src/CustomersList/TableList/style.css

```diff
@@ -0,0 +1,26 @@
+
+.customers_table{
```

Comment:
Why are using underscores? like `customers_table`. Class names should be with hyphens in case you want combine two words. Like `customers-table`. Change all the places